governance/governance.md

```diff
@@ -0,0 +1,255 @@
+# Governance
+
+According to the Business Dictionary, governance is defined as:
+
+> _Establishment of policies, and continuous monitoring
+> of their proper implementation, by the members of the
+> governing body of an organization. It includes the
+> mechanisms required to balance the powers of the
+> members (with the associated accountability), and
+> their primary duty of enhancing the prosperity and
+> viability of the organization_
+
+In open source, governance is described in the "governance
+model" document, defined by OSSWatch[^1] as:
+
+> _A governance model describes the roles that project participants 
+> can take on and the process for decision making within the project. 
+> In addition, it describes the ground rules for participation in the 
+> project and the processes for communicating and sharing within 
+> the project team and community_
+
+Usually the governance model is a written document containing:
+
+-  project goals
+-  work, management and collaboration infrastructures definition
+-  people roles and responsibilities definitions
+-  community support mechanisms
+-  decision making process policies description
+-  contribution process policies description
+-  monitoring policies and mechanisms description
+
+
+# Govenance Standards/Models
```

Comment:
```suggestion
## Governance Standards/Models
```

Spelling fix.

Also, should this be a H2>

Reply:
Well spotted typo. Thanks.

Not so sure about H2.

Reply:
I meant that given the rest of this page, this title should be at heading level 2 i.e. with a `##` in front.

I have updated my suggestion.

Reply:
Btw if you like you can merge `main` into your branch. Then you can use vale to check the spelling in this file (either locally or via the GitHub Action).